src/Storefront/Resources/app/storefront/src/plugin/header/search-widget.plugin.js

```diff
@@ -20,7 +20,6 @@ export default class SearchWidgetPlugin extends Plugin {
         searchWidgetCollapseClass: 'collapsed',
 
         searchWidgetDelay: 250,
-        searchWidgetMinChars: 3,
```

Comment:
This needs to be still here as default. when the search.html.twig is modified in the project it should still work without your changes

Reply:
Correct, I've re-added it and changed the Twig fallback also to 3